src/main/java/com/g4s8/ghman/App.java

```diff
@@ -51,11 +50,7 @@ private App() {
     @SuppressWarnings("PMD.CyclomaticComplexity")
     public static void main(final String[] args) throws Exception {
         final Options opts = new Options(new IterableOf<>(args));
-        final DataSource data = new FlywayDataSource(
```

Comment:
@scristalli how db schema will be set for the application? It will not work, if there is no structure for this application.

Reply:
@olenagerasimova same here, restored in commit 2e81d89ea35269f8cdc1f01c8e7b804811c762a8.